docs/api/robottelo.cli.rst

```diff
@@ -233,4 +238,4 @@ Submodules:
 :mod:`robottelo.cli.usergroup`
 ------------------------------
 
-.. automodule:: robottelo.cli.usergroup
\ No newline at end of file
+.. automodule:: robottelo.cli.usergroup
```

Comment:
That's really weird behavior. And github's diff contains this change too. @pondrejk could you please double check your commit's diff? If there's nothing changed then maybe better to exclude this change at all? I'm just afraid we can add some invisible unicode character here, as it happened before and caused issues for us.
Or maybe the root cause is different - there's no new line in the end of file. Mind adding it?